website/docs/gas.md

```diff
@@ -0,0 +1,33 @@
+# Gas fees update
+
+The gas fees update endpoint allows users to modify the current gas prices on a running devnet. This feature is particularly useful for testing purposes and for adjustments needed after forking to align with the forked network's gas prices.
```

Comment:
Let's capitalize: devnet -> Devnet

Reply:
Fixed in 88a74aa06b8be59925a6d0655efe3e63b78bde38